package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.7.6",
   "dependencies": {
     "httpreq": ">=0.4.22",
-    "underscore": "~1.7.0"
+    "underscore": "~1.12.1"
```

Comment:
I'd say `^1.12.1` would be even better.

Reply:
i am happy using "^" too - will change as soon as a maintainer requests.
But decided to go with the "~version" as the maintainer had choosen within their project for my initial PR.

Reply:
Is this going to be merged anytime soon?

Reply:
I am actually waiting for this fix too, thanks guys for addressing it!

Reply:
Waiting for this fix too